fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/Utils.kt

```diff
@@ -0,0 +1,102 @@
+package com.microsoft.fluentui.tokenized.segmentedcontrols
+
+import androidx.compose.foundation.interaction.InteractionSource
+import androidx.compose.foundation.interaction.collectIsFocusedAsState
+import androidx.compose.foundation.interaction.collectIsHoveredAsState
+import androidx.compose.foundation.interaction.collectIsPressedAsState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.graphics.Color
+import com.microsoft.fluentui.theme.token.ControlInfo
+import com.microsoft.fluentui.theme.token.ControlToken
+import com.microsoft.fluentui.theme.token.StateColor
+import com.microsoft.fluentui.theme.token.controlTokens.PillButtonInfo
+import com.microsoft.fluentui.theme.token.controlTokens.PillButtonTokens
+import java.security.InvalidParameterException
+
+@Composable
```

Comment:
Check if this file could be merged with others & put it in core module.

Reply:
Will do it in another PR with changes in all controls